conda/plugins/hookspec.py

```diff
@@ -15,9 +21,15 @@
     CondaVirtualPackage,
 )
 
+#: Name used for organizing conda hook specifications
 spec_name = "conda"
```

Comment:
```suggestion
spec_name = "conda"
"""Name used for organizing conda hook specifications"""
```

unless there was some reason for leaving this hidden from sphinx-autoapi